plugin/db/pg/pg.go

```diff
@@ -477,6 +480,7 @@ func (driver *Driver) SetupMigrationIfNeeded(ctx context.Context) error {
 		}
 
 		if !exist {
+			driver.l.Info("Try to creat bytebase database.")
```

Comment:
Don't make the same mistake as Ken Thompson

https://unix.stackexchange.com/questions/10893/what-did-ken-thompson-mean-when-he-said-id-spell-creat-with-an-e